### ISSUE TITLE
Admin only private messages

### ISSUE THREAD

**Issue by trichoplax** [codidact/qpixel#1390]
*This feature cannot be implemented until after #767 is merged.*

---

**Is your feature request related to a problem? Please describe.**
The new private messages to be introduced by #767 allow a moderator or admin to send a message to a user that is not visible to other users and not attached to a post. A moderator can see any messages sent by an admin or another moderator. There may be times when an admin wishes to send a private message that is not visible to moderators.

**Describe the solution you'd like**
A checkbox when an admin is drafting a private message, labelled something like "Also visible to moderators" which defaults to ticked but can be unticked to make the message visible to admins only (and the intended recipient).

I'm assuming that this functionality should only be available to admins, and that moderators will remain unable to send a private message without this being visible to other moderators.

**Comment by cellio** [comment]
Marking as low priority because it's a rare case and admins can contact users via email, but it would be better to have the activity (and the log of that activity) on-site, not in email.  We should do this (after the other branch merges), but it can wait a while.
